manager/drivers/provider.go

```diff
@@ -22,7 +22,7 @@ func (m *DriverProvider) NewSecretDriver(driver *api.Driver) (*SecretDriver, err
 	if m.pluginGetter == nil {
 		return nil, fmt.Errorf("plugin getter is nil")
 	}
-	if driver == nil && driver.Name == "" {
+	if driver == nil || driver.Name == "" {
```

Comment:
just a nitpick, shall we change the error to ? `driver specification is not set` as long as no caller is checking the actual error string

Reply:
Makes sense